racket/src/cs/rumble/system.ss

```diff
@@ -29,13 +29,17 @@
 
 (define os-symbol
   (case (reflect-machine-type)
-    [(a6osx ta6osx i3osx ti3osx arm64osx tarm64osx ppc32osx tppc32osx)
+    [(a6ios ta6ios arm64ios tarm64ios a6osx ta6osx i3osx ti3osx arm64osx tarm64osx ppc32osx tppc32osx)
      (if unix-style-macos? 'unix 'macosx)]
     [(a6nt ta6nt i3nt ti3nt arm64nt tarm64nt) 'windows]
     [else 'unix]))
 
 (define os*-symbol
   (case (reflect-machine-type)
+    [(a6ios ta6ios arm64ios tarm64ios)
+     (if unix-style-macos?
+         'unix
+         'ios)]
```

Comment:
At a minimum, I think this change should be consistent with BC:

https://github.com/racket/racket/blob/244535d8b41169ee69d077cd327090c12f9dd969/racket/src/bc/sconfig.h#L651-L657

IIUC the current behavior would be to return `'darwin` rather than `'unix`. (I think iOS is arguably Darwin, depending on where exactly you draw the line for "Darwin" between XNU and Mac OS.)

It might be more useful, though, to change both to return `'ios` even in `unix-style-macos?` builds: you can distinguish `unix-style-macos?` in any case with `(system-type 'os)` (no `*`), and `'ios` would let you distinguish iOS from Mac OS.